.githooks/pre-push

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+remote="$1"
+url="$2"
+
+z40=0000000000000000000000000000000000000000
+
+while read local_ref local_sha remote_ref remote_sha
+do
+    if [ "$local_sha" = $z40 ]; then
+        echo "Branch deleted. OK" && exit 0
+    fi
+
+    range="$remote_sha..$local_sha"
+    files=`git diff --name-only $range`
+
+    file_list_arg=""
+    for f in $files; do
+        fname=`basename $f`
+        file_list_arg="$file_list_arg $fname"
+    done
+
+    echo "Performing linter check: python tools/lint.py . --nproc=5 --file-list $file_list_arg"
+    python tools/lint.py . --nproc=5 --file-list $file_list_arg || exit 1
```

Comment:
Did you check if this work with a file in the dali/, include/ and dali_tf_plugin/ directories?

If yes, it should be fine.

Reply:
yes, it works for all of those